src/os_user.eliomi

```diff
@@ -33,14 +33,14 @@ val is_complete : t -> bool
 
 val emails_of_user : t -> string Lwt.t
 
-val add_activationkey : act_key:string -> int64 -> unit Lwt.t
+val add_activationkey : act_key:string -> int64 -> string -> unit Lwt.t
 val verify_password : email:string -> password:string -> int64 Lwt.t
 
 (** returns user information.
     Results are cached in memory during page generation. *)
 val user_of_userid : int64 -> t Lwt.t
 
-val userid_of_activationkey : string -> int64 Lwt.t
+val userdata_of_activationkey : string -> (int64 * string) Lwt.t
 (** Retrieve an userid from an activation key. May raise [No_such_resource] if
```

Comment:
Actually, the name of that function is a bit weird, because it's not so clear that it's returning the userid plus the email addr. of the user. Perhaps name it userid_and_email_of_activationkey or else return more complete userdata in a record?